### PR TITLE
[event-monitor] Add subscription spinner & disable cometd debug

### PR DESCRIPTION
**Pull request by AntoineLeleu-Salesforce**
## Describe your changes
Add Spinner during the subscription.
Disable cometd debug.

## Checklist before requesting a review
- [x] I have read and understand the [Contributions section](https://github.com/tprouvot/Salesforce-Inspector-reloaded#contributions)
- [x] Target branch is releaseCandidate and not master
- [x] I have performed a self-review of my code
- [x] I ran the [unit tests](https://github.com/tprouvot/Salesforce-Inspector-reloaded#unit-tests) and my PR does not break any tests
- [ ] I documented the changes I've made on the [CHANGES.md](https://github.com/tprouvot/Salesforce-Inspector-reloaded/blob/master/CHANGES.md) and followed actual conventions
- [ ] I added a new section on [how-to.md](https://github.com/tprouvot/Salesforce-Inspector-reloaded/blob/master/docs/how-to.md) (optional)

### CODE REVIEW

File: addon/event-monitor.js [hunk 1]
```diff
@@ -229,7 +231,7 @@ class App extends React.Component {
         Authorization: "Bearer" + model.sessionId
       },
       appendMessageTypeToURL: false,
-      logLevel: "debug"
+      //logLevel: "debug"
```
Comment (tprouvot) [inline addon/event-monitor.js:234]:
```suggestion
```

**Review by tprouvot — COMMENTED** [review]
Could you remove the commented code ?

**Comment by tprouvot** [comment]
@AntoineLeleu-Salesforce great ! My intention was to add the spinner later, thank you for doing it now 👏

File: addon/event-monitor.js (hunk 1 above)
Comment (AntoineLeleu-Salesforce) [reply addon/event-monitor.js]: @tprouvot done i

**Review by tprouvot — APPROVED** [review]

**Pull request merged**